Classes/View Controllers/UINavigationItem+TitleSubtitle.swift

```diff
@@ -45,6 +64,9 @@ extension UINavigationItem {
         label.sizeToFit()
 
         titleView = label
+        if let accessibilityLabel = accessibilityLabel { // prevent from setting to nil if we don't provide anything, use default instead
```

Comment:
Now that I am reading this comment, I think it can be more clear. Do you understand what I am trying to say here? 😁

Reply:
Ha, good enough for me. I can follow the code, could even just delete the comment 🤷‍♀️